cvat/apps/documentation/user_guide.md

```diff
@@ -722,7 +722,12 @@ Go to the next/previous frame with a predefined step. Shortcuts:
 
   ![](static/documentation/images/image037.jpg)
 
-Go to the next/previous frame (the step is 1 frame). Shortcuts: ``D`` — previous, ``F`` — next.
+The button to go to the next / previous frame has the customization possibility. To customize, right-click on the button and select one of three options:
+1. The default option - go to the next / previous frame (the step is 1 frame).
+2. Go to the next / previous frame which corresponds to the set filter (read the [filter](#filter) section to know the details how to use it).
```

Comment:
I would say: Go to the next / previous frame that has any objects (in particular filtered) ...